spec/services/workflow_service/property_assessment_spec.rb

```diff
@@ -123,7 +117,7 @@ module WorkflowService # rubocop:disable Metrics/ModuleLength
             expect(ap2.property_disregard).to eq 0.0
             expect(ap2.assessed_capital_value).to eq 221_900.0
 
-            md = particulars.response.details.capital.property.main_dwelling
+            md = result.main_home
```

Comment:
If `main_dwelling` is now `main_home`, should `md` now be `mh`?

Reply:
👍